src/main/java/com/minecolonies/items/ItemSupplyChestDeployer.java

```diff
@@ -85,6 +86,10 @@ public boolean onItemUse(ItemStack stack, EntityPlayer playerIn, World worldIn,
         {
             spawnShip(worldIn, pos, playerIn, facing);
             stack.stackSize--;
+            
+            // Trigger Achievement
```

Comment:
This comment seems kinda pointless, its just the name of the method you're calling.